collect/usace/wcds.py

```diff
@@ -54,7 +54,7 @@ def get_water_year_data(reservoir, water_year, interval='d'):
 
     # Convert to date time object
     df.set_index('ISO 8601 Date Time', inplace=True)
-    df.index = pd.to_datetime(df.index)
+    df.index = pd.to_datetime(df.index.str.replace('T24:', ' '))
```

Comment:
Using a different format, to illustrate, but I believe this change would convert the time 
2022-09-29 **24:00**
to 
2022-09-29 **00:00**

Reply:
If you have a better idea for how to do this i'm open...
[b736cd2](https://github.com/MBKEngineers/collect/pull/108/commits/b736cd240ba2a8801384b69dcbc240b22d75fb89)